### PR TITLE
Add new calculator.css and calculator.js files

### PR DESCRIPTION
**Pull request by JasemDuncan**
For this milestone, a new javascript was created to make the layout of the project. Also, it is render from App.js

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      <button type="button">1</button>
+      <button type="button">2</button>
+      <button type="button">3</button>
+      <button type="button" className="orange-buttton">+</button>
+      <button type="button" className="span-two">0</button>
+      <button type="button">.</button>
+      <button type="button" className="orange-buttton">=</button>
+    </div>
+  </div>
+);
```
Comment (omar-labana) [inline src/components/Calculator.js]: - [x] Kindly, update the `Calculator` compoenent to be a class-based component.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @JasemDuncan, 

Excellent work so far! 👌
You have implemented almost all the required features. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
- [x] Please remove the react logo from the `src` directory.


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+//       <button type="button" className="orange-buttton">-</button>
+//       <button type="button">1</button>
+//       <button type="button">2</button>
+//       <button type="button">3</button>
+//       <button type="button" className="orange-buttton">+</button>
+//       <button type="button" className="span-two">0</button>
+//       <button type="button">.</button>
+//       <button type="button" className="orange-buttton">=</button>
+//     </div>
+//   </div>
```
Comment (sinansevgi) [inline src/components/Calculator.js:68]: [Optional]
- [ ] You should keep your codebase clean. Please remove commented-out code blocks. You can see your old code with git history.

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @JasemDuncan, 

The calculator component is a class-based component now :clap:
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**